<!-- help.html File -->
<div class="container">
  <!-- <h1 class="title fit-h1">Who is Intelligentsia?</h1> -->
  <div class="row">
      <h1  class="title fit-h1">Need Help? We Got You Covered.</h1>
  </div>
</div>
<div class="container">
  <!-- <h1 class="title fit-h1">Who is Intelligentsia?</h1> -->
  <div class="row">
      <h2 class="title fit-h1">How to Get Started:</h2>
      <p> JUMPn Analysis relies on the user to upload a data file containing in one column
        a list of genes, while all other columns of the table possess the gene expression data
        for each gene across the various conditions </p>
  </div>
</div>

<div class="container">
  <!-- <h1 class="title fit-h1">Who is Intelligentsia?</h1> -->
  <div class="row">
      <h2 class="title fit-h1">Input File Specifications:</h2>
      <p> If you're receiving an error right after submitting your data for analysis, it is likely
      because the input file is not in a supported format. At the Commence Analysis Page, once having clicked
    the "Begin Analysis" button, the user is prompted to upload an input file. The program exclusively allows
  for .xlsx files, so right off the bat, this may be one of your issues. If your file is already in .xlsx format,
your issue may occur in the specification of additional upload information. At the additional upload information
page, one parameter asks the user to specify the type of JUMP file used for the input. In this parameter, the program
wants to know what type of file is to be expected. The DE_prot file is the default as most users execute JUMPn Analysis
after identifying differentially expressed proteins from their data set. The most common type of DE_prot file lacks the one
line summary so this is the default input file. A variety of other files are supported, including uni_prot_quan files and
uni_prot_hpc files. These are typically much larger, and both have a four line summary. Finally, an additional DE_prot file
  has a one line summary, and are relatively common. If your input file does not come from JUMP, then you can create your
  own .xlsx file. If none of these have solved your issue, then the most likely cause of your problems is occurring as a
  consequence of your column headers. It is very important that the gene name column has the header "GN" and each of your
  TMT columns begins with "sig" and specify this in the input section.

 </p>
  </div>
</div>
<div class="item">
  <!-- I need to find a way to insert a cluster image here @ line 32 -->
  <img style="position:relative; margin-left:auto; margin-right:auto;" src="images/example_output.png" alt="5">
</div>
<!-- Need to Include Documentation to Application Use Here -->
<div class="container">
  <!-- <h1 class="title fit-h1">Who is Intelligentsia?</h1> -->
  <div class="row">
      <h2 class="title fit-h1">Specifying Output Locations</h2>
      <p> It is critical that the user specifies an output folder name. This folder will be
      created and all data will be exported to this location. It is also critical that the
      output folder is not a folder already in existence, as this can cause major issues with
    the execution of the analysis </p>
  </div>
</div>
<!-- Need to Include Documentation to Application Use Here -->

<!-- Need to Include Documentation to Application Use Here -->
<div class="container">
  <!-- <h1 class="title fit-h1">Who is Intelligentsia?</h1> -->
  <div class="row">
      <h2 class="title fit-h1">Co-Expression Clustering Parameters</h2>
      <p> JUMPn Analysis affords the user a variety of options for controlling statistical parameters
      regarding the clustering analysis and WGCNA algorithm. Many of these can be used at their default
    levels to return a large aggregate of statistically significant results. However, if you'd like to explore
  your data a bit more at the co-expression cluster level, then this is a good area to control these features.
First of all, WGCNA identifies modules of a minimum size. Increasing the minimum module size will increase the significance
of your results, as only the larges co-expression clusters will be identified, but you may miss some smaller expression
clusters that otherwise would generally be overlooked. We traditionally hold the Pearson Correlation at .7, which is the minimum
correlation between gene expression which must be met in order to be included in the expression cluster. The user can raise
the Pearson correlation value to receive more statistically significant co-expression clusters, or lower this value for broader clusters.
The minimum R squared value for WGCNA is another useful parameter control which allows the user to control the stringency of
WGCNA driven modularization.  </p>
  </div>
</div>
<!-- Need to Include Documentation to Application Use Here -->
<div class="container">
  <!-- <h1 class="title fit-h1">Who is Intelligentsia?</h1> -->
  <div class="row">
      <h2 class="title fit-h1">Protein-Protein Interaction Parameters</h2>
      <p> After executing co-expression clustering by WGCNA, the JUMPn program overlays the relevant protein protein interactions
      and identifies modules based on their physical interactions with other components in your gene list. The user has the option
    of skipping the PPI overlay, but this is not advised as the user will not receive a modularized cluster. In order for the cluster to be
  modularized, it must first have a minimum size in the cluster, which the user must set. Then the program will identify smaller modules
which also must meet a minimum number of proteins to be considered a module. Raising this parameter will likely yield less modules, but these
modules will certainly exhibit more complex protein interactions. The user can also control how many proteins appear in the network visualization
using the minimum number of proteins to be shown in the network analysis visualization</p>
  </div>
</div>
<!-- Need to Include Documentation to Application Use Here -->
